frontend/packages/ceph-storage-plugin/src/__mocks__/independent-mode-dashboards.ts

```diff
@@ -0,0 +1,67 @@
+export const DashboardItems = {
+  watchURL: {
+    url: 'foo',
+  },
+  stopWatchURL: 'foobar',
+  watchPrometheus: {
+    query: 'fooQuery',
+  },
+  stopWatchPrometheusQuery: {
+    query: 'fooQuery2',
+  },
+  watchAlerts: '',
+  stopWatchAlerts: '',
+  urlResults: 'foo',
+  prometheusResults: {
+    // eslint-disable-next-line no-new-func
+    getIn: new Function(),
+  },
+  notificationAlerts: 'foo',
+  watchK8sResource: 'foo',
+  stopWatchK8sResource: 'foo',
+};
+
+export const dropDownItems = {
+  Pods: 'Pods',
+  Projects: 'Projects',
+  'Storage Classes': 'Storage Classes',
+};
+
+export const headerLink =
```

Comment:
`headerLink` and `dropDownItems` are treated as expected test data.

Therefore, I'd suggest naming them `expectedDropDownItems` and `expectedHeaderLink` respectively.